crates/serde/src/json_u256.rs

```diff
@@ -1,5 +1,6 @@
 //! Json U256 serde helpers.
 
+#[allow(unused_imports)]
```

Comment:
Seems the clippy warnings are not correct, for no_std must import `String`/`ToString`

Reply:
you can feature gate these imports instead:

see also 

https://github.com/alloy-rs/alloy/blob/main/crates/eips/src/eip2930.rs#L7-L8